guidelines/terms/21/single-pointer.html

```diff
@@ -1,7 +1,5 @@
 <dt><dfn>single pointer</dfn></dt>
 <dd>
-   	
-   <p>pointer input that operates with one point of contact with the screen, including single taps and clicks, double-taps and clicks, long presses, and path-based gestures</p>
-  				
+   <p>an input – such as a mouse, stylus, or finger on a touch screen – that only targets a single point on the screen (as opposed to multi-pointer/multi-touch scenarios)</p>
```

Comment:
```suggestion
   <p>an input – such as a mouse, stylus, or finger on a touch screen – that only targets a single point on the screen (in contrast to multi-pointer/multi-touch scenarios)</p>
```

Reply:
could add something like 'or an assistive device using the pointer API' or something

Reply:
"vector" input?

Reply:
or specific coordinate

Reply:
as with the discussion around other alternative input methods, like sip and puff, these methods like "AT firing synthetic pointer events" all boil down to still being mouse/touch/stylus, so in the interest of brevity/readability, i'd suggest not starting to list all other ways in which these events may get fired (this is in relation to the "assistive device..." comment).

as for "vector" input (an input with a magnitude/direction), that's not really relevant here (and risks confusing matters further when we then talk about dragging movements etc)

Reply:
problem with "coordinate" is that you can't just say "a coordinate"...you need "a set of coordinates", "a pair of coordinates", or just plural "coordinates", which then introduces ambiguity again about only being a single point. and then also precludes things like VR/AR where the coordinates may be in 3D space